filter/filter_test.go

```diff
@@ -15,12 +15,31 @@ func TestIsExcluded(t *testing.T) {
 		f := filter.Filter{Exclude: []string{"a", "b"}}
 		assert.True(t, f.IsExcluded("a"))
 	})
+	t.Run("True", func(t *testing.T) {
+		f := filter.Filter{Exclude: []string{"a", "b"}}
+		assert.True(t, f.IsExcluded("a", "b"))
+	})
```

Comment:
Could you add (on `IsExcluded` and `IsIncluded`) a test like `f.IsExcluded("a", "c")`

Reply:
test added